client-src/elements/chromedash-typeahead_test.ts

```diff
@@ -199,13 +202,13 @@ describe('chromedash-typeahead-item', () => {
         prefix="aVal"
       ></chromedash-typeahead-item>
     `);
-    const valueEl = component.shadowRoot.querySelector('#value');
-    assert.include(valueEl.innerHTML, 'aVal');
-    assert.include(valueEl.innerHTML, 'ue');
-    const bold = component.shadowRoot.querySelector('b');
-    assert.include(bold.innerHTML, 'aVal');
-    const docEl = component.shadowRoot.querySelector('#doc');
-    assert.include(docEl.innerHTML, 'Docs about it');
+    const valueEl = component.shadowRoot!.querySelector('#value');
+    assert.include(valueEl!.innerHTML, 'aVal');
```

Comment:
Can we avoid the repeated `!`s below by adding:
`assert.exists(valueEl);`?

Reply:
yeah that's a good idea, rendorRoot is not applicable to those so I used ! in the first place